alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/GrpcStartable.java

```diff
@@ -202,4 +208,55 @@ private OptionalInt findUnusedPort(GrpcServerConfig serverConfig) throws IOExcep
       throw error[0];
     }
   }
+
+  /**
+   * https://netty.io/wiki/native-transports.html
+   *
+   * RHEL/CentOS/Fedora:
+   * sudo yum install autoconf automake libtool make tar \
+   *                  glibc-devel libaio-devel \
+   *                  libgcc.i686 glibc-devel.i686
+   * Debian/Ubuntu:
+   * sudo apt-get install autoconf automake libtool make tar \
+   *                      gcc-multilib libaio-dev
+   *
+   * brew install autoconf automake libtool
+   * */
+  private Class<? extends ServerChannel> selectorServerChannel() {
+    Class<? extends ServerChannel> channel = NioServerSocketChannel.class;
+    if (serverConfig.isNettyTransport()) {
+      if (OSInfo.isLinux()) {
```

Comment:
It's better to add some Info log to tell user which kind of ServerSocketChannel that Alpha choices.

Reply:
I have marked the location of the log output below.

Reply:
OK, I missed it.